src/Open3D/Visualization/Rendering/Filament/FilamentView.h

```diff
@@ -26,12 +26,12 @@
 
 #pragma once
 
-#include "Open3D/Visualization/Rendering/View.h"
+#include <filament/Color.h>
 
```

Comment:
delete blank line?

Reply:
clang-format inserted the blank line. check-style fails without it.

Reply:
this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.